add-ons/htpasswd-identity-provider/htpasswd-identity-provider.addon.remove

```diff
@@ -0,0 +1,19 @@
+# Name: htpasswd-identity-provider
+# Description: Reverts to default minishift authentication configuration
+# Url: https://docs.openshift.org/3.9/install_config/configuring_authentication.html#AllowAllPasswordIdentityProvider
+# Var-Defaults: USERNAME=developer,MINISHFIT_DATA_HOME=/var/lib/minishift,CONFIG_LOCATION=openshift.local.config/master,MASTER_CONFIG_FILE=master-config.yaml
```

Comment:
I think `USERNAME` can be removed from here, no use.

Reply:
no lets be there, if the user does not provide it first time application then we need to use `developer`

Reply:
@kameshsampath in addon remove we don't need any user details so can be removed.